x/inter-tx/keeper/msg_server.go

```diff
@@ -85,7 +85,8 @@ func (k msgServer) Delegate(goCtx context.Context, msg *types.MsgDelegate) (*typ
 		Data: data,
 	}
 
-	_, err = k.icaControllerKeeper.TrySendTx(ctx, chanCap, portID, packetData)
+	timeoutTimestamp := ^uint64(0) >> 1
```

Comment:
I would rather not use this if there's an alternative. I tried testing with: `uint64(ctx.BlockTime().Add(time.Hour).Unix())` but I saw no packets being relayed.

Reply:
Maybe we can just add a comment in the short term explaining

Reply:
Done 👍